packages/icon/README.md

```diff
@@ -83,14 +85,51 @@ const SomeComponent = () => (
 We also export the default icon set for you! If you want to include our glyphs with your custom glyphs, you can do something like this:
 
 ```js
-import { createIconComponent, glyphs } from '@leafygreen-ui/Icon';
+import { createGlyphComponent, createIconComponent } from '@leafygreen-ui/icon';
 
 const myGlyphs = {
   ...glyphs,
-  MyCustomGlyph: () => <svg />,
+  MyCustomGlyph: createGlyphComponent('MyCustomGlyph', () => <svg />),
 };
 
 const MyIconComponent = createIconComponent(myGlyphs);
 ```
 
 **Note:** Glyph has a static property, `isGlyph`, that enables checking whether or not a component is a LeafyGreen glyph.
+
+## Returns
+
+A custom `Icon` component, that includes any custom `glyph`s
+
+## Properties
+
+| Prop          | Type                                | Description                                                | Default |
+| ------------- | ----------------------------------- | ---------------------------------------------------------- | ------- |
+| `GlyphObject` | `Record<string, LGGlyph.Component>` | A dictionary record of all the named icons in the icon set |         |
+
+# createGlyphComponent
+
+Ensures a custom glyph accepts the same props, and behaves the same, as a built-in Leafygreen icon.
```

Comment:
This doesn't make the component "behave the same", just accept the same props. We need to recommend spreading props onto their `<svg>` element, and that all fills are set to `currentColor`. We should probably also recommend that the icons have all strokes outlined and such too.